miio/airhumidifier.py

```diff
@@ -126,3 +161,27 @@ def set_buzzer(self, buzzer: bool):
             return self.send("set_buzzer", ["on"])
         else:
             return self.send("set_buzzer", ["off"])
+
+    def set_child_lock(self, lock: bool):
+        """Set child lock on/off."""
+        if lock:
+            return self.send("set_child_lock", ["on"])
+        else:
+            return self.send("set_child_lock", ["off"])
+
+    def set_target_humidity(self, humidity: int):
+        """Set the target humidity."""
+        if humidity not in [40, 50, 60, 70, 80]:
+            raise AirHumidifierException(
+                "Invalid target humidity: %s" % humidity)
+
+        return self.send("set_limit_hum", [humidity])
+
+    def set_favorite_level(self, level: int):
+        """Set favorite level."""
+        if level < 30 or level > 85:
+            raise AirHumidifierException("Invalid favorite level: %s" % level)
```

Comment:
Is there a link between this level and target humidity (which allows only specific values)?

Reply:
No. It's more like a "fan speed".

Reply:
I'm waiting for some details currently: https://community.home-assistant.io/t/xiaomi-humidifier-support/35672/26

Reply:
Ok, feel free to merge when you are ready :+1: